bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/update/update.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="openwebnet:bus_alarm_zone">
+		<instruction-set targetVersion="1">
+			<add-channel id="timestamp">
+				<type>openwebnet:zoneAlarmTimestamp</type>
+				<label>Zone Alarm Timestamp</label>
```

Comment:
No need to set a label, it will be taken from the channel type.
Requires openhab/openhab-core#3576

Reply:
OK. Label tag removed